tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -130,6 +152,17 @@ func TestMain(m *testing.M) {
 	}
 
 	log.Printf("Test log: %s\n", logFile)
+
+	// tmp dir setup
+	tmpDir, err := os.MkdirTemp(mntDir, "tmpDir")
+	if err != nil {
+		printAndExit(fmt.Sprintf("Mkdir at %q: %v", mntDir, err))
+	}
+
+	// tmp file setup
+	fileName = path.Join(tmpDir, "tmpFile")
+	err = os.WriteFile(fileName, []byte("line 1\nline 2\n"), 0666)
```

Comment:
we should stop test execution if there is an error in writing to file.

Reply:
Done